src/sentry/issues/json_schemas.py

```diff
@@ -11,6 +11,7 @@
         "timestamp": {"type": "string", "format": "date-time"},
         "received": {"type": "string", "format": "date-time"},
         "title": {"type": "string", "minLength": 1},
+        "level": {"type": "string", "minLength": 1},
```

Comment:
can we move this below the comment? and alphabetical order everything

Reply:
These are already kind of unordered, is the idea to sort by required/key?

Reply:
lol it was in order until you changed it. whatever you want tho

Reply:
but yeah required on top then non-required at bottom. both sorted a-z

Reply:
`received` was already out of order though? anyway, happy to change it

Reply:
My thinking there was that we were generating that value so it would go at the bottom. Whatever is fine.